src/recommendation/src/main/java/com/redhat/developer/demos/recommendation/RecommendationVerticle.java

```diff
@@ -12,15 +12,22 @@
 
 public class RecommendationVerticle extends AbstractVerticle {
 
-    private static final String RESPONSE_STRING_FORMAT = "recommendation v1 from '%s': %d\n";
```

Comment:
We shouldn't be using these ones, we should be using the quarkus ones from the istio-tutorial.  We will move those across at some point but lets focus on the docs while we do this.

Reply:
The current java code causes the curl line to always respond with v1 even when it is hitting v2 or v3. My change corrects that.

Reply:
We don't use this version though, we use the quarkus one from the istio-tutorial.